src/pages/Settings/FirstMFASetup.vue

```diff
@@ -5,7 +5,7 @@
 		</p>
 		<kv-button
 			class="tw-w-full"
-			:to="`/ui-login?doneUrl=${encodeURIComponent('/login?authLevel=mfa&doneUrl=settings/security/mfa')}`"
+			:to="`/ui-login?doneUrl=${encodeURIComponent('/login?authLevel=mfa&doneUrl=settings/security')}`"
```

Comment:
This seems like it's going to take you out of the flow in a sense as the current destination will show the status of the 2fa you just setup... is there a need to change this?

Reply:
I suppose not, since the filter I added will disable that page from removing everything. The `settings/security` page does show the status as enabled, but I can revert this line now.